runtime/src/snapshot_utils.rs

```diff
@@ -1636,7 +1649,7 @@ pub fn snapshot_bank(
     hash_for_testing: Option<Hash>,
     snapshot_type: Option<SnapshotType>,
 ) -> Result<()> {
-    let snapshot_storages = get_snapshot_storages(root_bank, snapshot_type);
+    let snapshot_storages = get_snapshot_storages(root_bank, None);
```

Comment:
I think passing `None` here will end up with an empty `SnapshotStorages`.

Since `get_snapshot_storages()` only exists to make getting either full or incremental snapshot storages from the bank easier, that function can be changed to reflect the new desired behavior. So the `snapshot_type` parameter can be removed, and the impl can just call `bank.get_snapshot_storages(None)`.

I'd like to keep the datapoint, but that can now be simplified to just `snapshot-storages-count` and `snapshot-storages-time-ms`.